common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -496,6 +515,16 @@ def _special_exam_student_view(self):
                     banner_text = _("This exam is hidden from the learner.")
                     return banner_text, special_exam_html
 
+    def _hidden_content_banner_text(self, course):
```

Comment:
is it worth testing the different text here?

Reply:
Eh, it's such a minor difference and only for staff, so I'm inclined to not bother.